src/components/Geo/index.jsx

```diff
@@ -1,10 +1,36 @@
 import React from 'react'
+import ReactDOM from "react-dom";
 import { Helmet } from 'react-helmet';
 import MapPlot from './MapPlot';
 import SelectionInfo from './SelectionInfo';
+import {
+    ExternalLink,
+    helpIcon
+} from '../../common/Helpers';
+import styles from '../../styles/collapse.css';
```

Comment:
styles should go in [tailwind.css](https://github.com/serratus-bio/serratus.io/blob/a862c4d14a4b81d1a760ef383226dfd38c3c9833/src/styles/tailwind.css), this way you won’t need the import here.